### PR TITLE
Update AboutUs

### PR DESCRIPTION
**Pull request by tanweijie123**
Included Roles and responsibilities for everyone in the team

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -1,45 +1,59 @@
----
-layout: page
-title: About Us
```
Comment (benclmnt) [inline docs/AboutUs.md:3]: don't change this. It is used by the jekyll

Comment (benclmnt) [inline docs/AboutUs.md]: Don't remove this part. It is used by Jekyll

**Review by benclmnt — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md [hunk 2]
```diff
@@ -3,43 +3,64 @@ layout: page
 title: About Us
 ---
 
+<head> <title> About Us </title> </head>
+
+# About Us
+---
```
Comment (benclmnt) [inline docs/AboutUs.md]: Perhaps, if you add this then we will have double title on the page (?)

**Review by benclmnt — CHANGES_REQUESTED** [review]
Other than that issue, overall looks good to merge for me

File: docs/AboutUs.md (hunk 2 above)
Comment (tanweijie123) [reply docs/AboutUs.md]: The <title> will set it as tab name, while the # About Us is the in the webpage.

**Review by benclmnt — APPROVED** [review]
looks good to merge

**Pull request merged**